examples/semi_supervised/plot_label_propagation_versus_svm_iris.py

```diff
@@ -54,14 +64,16 @@
 titles = ['Label Spreading 30% data',
           'Label Spreading 50% data',
           'Label Spreading 100% data',
+          'Self-training 30% data',
+          'Self-training 50% data',
           'SVC with rbf kernel']
 
 color_map = {-1: (1, 1, 1), 0: (0, 0, .9), 1: (1, 0, 0), 2: (.8, .6, 0)}
 
-for i, (clf, y_train) in enumerate((ls30, ls50, ls100, rbf_svc)):
+for i, (clf, y_train) in enumerate((ls30, ls50, ls100, st30, st50, rbf_svc)):
```

Comment:
Maybe put st30 ad st50 at the top: this way we could easily visually compare st30 with ls30 and st50 with ls50.